app/src/constants/baseSEO.ts

```diff
@@ -0,0 +1,26 @@
+import { landingContent } from "@/constants/content";
+
+export default {
+  description: landingContent.subtitle,
+  openGraph: {
+    type: "website",
+    locale: "en_IE",
+    url: "https://montelearn.monterail.com/",
```

Comment:
This should be env or const at least but did not decide where to put it, suggestions? 🤔

Reply:
I think it's fine there 😄 
Alternatively, we can create a file for project URLs in constants directory and have it there, with env var we would need to engage TKO